README.md

```diff
@@ -86,15 +86,36 @@ Note that non-default networks require extra [firewall setup](https://cloud.goog
   external provisioner can be used to provide similar functionality for
   Kubernetes clusters.
 
+## Deploying the Driver
```

Comment:
do we need to indicate anywhere this is for Linux driver?

Reply:
Good point, I'll add under future features.